optima/results.py

```diff
@@ -100,9 +102,49 @@ def __repr__(self):
 
 
 
+class BOC(object):
+    ''' Structure to hold a budget and outcome array for geospatial analysis'''
+    def __init__(self, name='unspecified', x=None, y=None, objectives=None):
```

Comment:
@davidkedz of course it's not urgent, but it would be nice to rename x and y something like budget and outcome, or totalspends and epioutcomes, or anything else that would give a clue about what they should be.